.github/workflows/publish-noir-js.yml

```diff
@@ -17,9 +17,13 @@ jobs:
       - name: Install jq
         run: sudo apt-get install jq
 
+      - name: Setup toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: 1.67.0
+
       - name: Install cargo and wasm-bindgen
         run: |
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
           source $HOME/.cargo/env
           cargo install -f wasm-bindgen-cli --version 0.2.86
```

Comment:
```
      - name: Install wasm-bindgen-cli
        uses: taiki-e/install-action@v2
        with:
          tool: wasm-bindgen-cli@0.2.86
```

Reply:
This removes the need for bumping the MSRV.

Reply:
Revised